tools/export/nb/__init__.py

```diff
@@ -325,9 +325,11 @@ def get_netbeans_file_list(file_list):
             if cur_dir and prev_dir != cur_dir:
                 # evaluate all matched items (from current and previous list)
                 matched = []
-                for element in dir_list:
-                    if element in prev_dir_list:
-                        matched.append(element)
+                # Compare the Element in Previous Dir with the Elements in Current Dir
+                # and add the equal Elements to the match-List
+                for elem_prev_dir, elem_cur_dir in zip(prev_dir_list, dir_list):
+                    if elem_prev_dir == elem_cur_dir:
+                        matched.append(elem_cur_dir)
```

Comment:
The use of `elem` in these variable names dose not add any meaning for me. I would like to seem them removed, but I won't block this PR for it.

Reply:
If I remove the prefix  `elem` it will not work as expected, because the variables prev_dir and cur_dir are already defined in scope.

Reply:
That's part of why I would like another name: the current name is almost a name clash. Like I said, not blocking just a wish. Let me know if you come up with something.